docs/Cerberus.md

```diff
@@ -0,0 +1,39 @@
+# Cerberus
+
+[Cerberus](https://www.cl.cam.ac.uk/~pes20/cerberus/) offers executable semantics for a substantial fragment of C and
+CHERI-C languages. It is implemented via an elaboration into a simpler Core language, which is displayed as the compiler
+output in the Compiler Explorer. Evaluation of C programs (execution) is also supported.
+
+## Prerequisites
+
+The easiest way to install both the Cerberus and Cerberus-CHERI compilers is by using Docker:
+
+`docker pull vzaliva/cerberus-cheri`
+
+Then, for example, you can print the _Core_ elaboration for `test.c` using ISO C semantics:
+
+`docker run -v $HOME/tmp:/mnt -it vzaliva/cerberus-cheri cerberus --pp=core --exec /mnt/test.c`
+
+## Configuration
+
+The file `etc/config/c.defaults.properties` defines a group of two compilers: 'cerberus' for ISO C and 'cerberus-cheri'
+for CHERI-C. It assumes that the corresponding executables are in the path.
```

Comment:
> It assumes that the corresponding executables are in the path.

We avoid this in general as it leads to surprises, but I understand why it's useful for a local installation only.